tests/testthat/test_mlr_measures.R

```diff
@@ -28,11 +28,19 @@ test_that("mlr_measures", {
     })
     expect_number(perf, na.ok = "na_score" %in% m$properties)
 
+    if (key == "surv.graf") {
+      expect_equal(unname(perf), 1.070536, tolerance = 1e-05)
+    }
+
+    # test measures with squared-errors
```

Comment:
From previous chat, all these tests I did them to check that the graf score remains the same if the input to `weighted_survival_score()` is `prediction$distr` and not `prediction$data$distr` in case of a survival array and `Arrdist` conversion (and it was okay). You reverted that change - mentioning that 
1) might break parametric models (can't really say, which learner you have in mind, I can test?)
2) inconsistent with all other measures (hm, but all other scores that call `weighted_survival_score()` use `prediction$distr`!)

Reply:
So after the latest updates, the only score that fails with survival 3d arrays in the `PredictionSurv` object is `surv.graf`. This line is hte one that causes the problem => https://github.com/bblodfon/mlr3proba/blob/main/R/integrated_scores.R#L32.

Reply:
Sorry I actually meant more that this sort of test is not really useful, i.e., testing against a single hardcoded number. This sort of test is only useful as 'sanity' checks to make sure something fundamental in the code hasn't broken. So here I'd just recommend checking you can run it silently (i.e., without error) or checking `Arrdist` result vs. `WeightedDisc`/`Matdist` (but in that case your comparing it to the dynamic output from the same test using `Matdist`, but to be honest this test is redundant because if we're satisfied extraction works correctly then this doesn't add anything)

Reply:
I will remove these tests. I also changed a bit the code on `weighted_survival_score()` to make it work when `pred$data$distr` is a survival 3d array.

Reply:
Tests removed